tests/dist/python/run_dist_objects.py

```diff
@@ -695,12 +695,20 @@ def test_dist_dataloader(g):
         "DistDataLoader": test_dist_dataloader,
     }
 
-    target = os.environ.get("DIST_DGL_TEST_OBJECT_TYPE", "")
-    if target not in target_func_map:
+    targets = os.environ.get("DIST_DGL_TEST_OBJECT_TYPE", "")
+    targets = targets.split(',') if targets else []
```

Comment:
cannot handle `"test_0, test_1,test_2"` which has unexpected white space. but I think it's common to hit such case.

Reply:
Done